packages/genui/playground/src/pages/chat/ChatPage.css

```
@@ -1591,3 +1601,104 @@
+
+.chatArtifactCodeBlock {
+  max-height: 360px;
+  margin: 0;
+  overflow-x: hidden;
+  border-top: 0;
+  border-radius: 0 0 var(--geist-radius-md) var(--geist-radius-md);
+  padding: 8px 10px;
+  line-height: 1.55;
+  white-space: pre-wrap;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)